components/common/src/command/package/install.rs

```diff
@@ -370,7 +370,7 @@ impl<'a> InstallTask<'a> {
     fn new(
         install_mode: &'a InstallMode,
         local_package_usage: &'a LocalPackageUsage,
-        url: &str,
+        bldr_url: &str,
```

Comment:
Very nice catch--we do tend to call that `bldr_url` in most other places. One day, perhaps also not a direct string representation too.

Reply:
I had to change this in _so_ many places and I'm relatively certain I missed it in a few as well.